generic/cloudbuild.yaml

```diff
@@ -1,4 +1,4 @@
-# gcloud builds submit --config=./generic/cloudbuild.yaml . --substitutions=_VERSION=0.9.0
+# gcloud builds submit --config=./generic/cloudbuild.yaml . --substitutions=_VERSION=0.10.0
```

Comment:
You specify version 0.10.0 here... but further down and in the pom file it still says version 0.9.0